source/standards/logging.html.md.erb

```diff
@@ -1,172 +1,130 @@
-      - example: `/dir1/dir2/resource`
-  - `query_string`
-      - type: string
-      - meaning: HTTP query string, *excluding* initial question mark
-        character
-      - example: `lang=en&q=hello`
-  - `referrer` (not `referer`)
-      - type: string
-      - meaning: HTTP Referer header contents
-  - `response_code`
-      - type: integer
-      - meaning: HTTP status code
-      - If the status code does not exist for some reason (say,
-        because the connection was closed or reset before the status
-        was sent) then a special numerical value can be used, or the
-        field can be dropped.
-      - examples: `200`, `404`
-  - `url`
-      - type: string
-      - meaning: requested URL, relative to server root, including
-        query string if provided
-      - example: `/dir1/dir2/resource?lang=en&q=hello`
-  - `user_agent`
-      - type: object
-      - meaning: parsed fields representing user agent.  Generated
-        using logstash `useragent` filter, or elasticsearch
-        ingest-user-agent plugin.
-  - `user_name`
-      - type: string
-      - meaning: user name of authenticated user.  In particular, this
-        refers to HTTP authentication such as basic auth, rather than
-        application-level authentication.
-
-Example:
-
-```json
-{
-  "@version": 1,
-  "@timestamp": "2017-07-11T15:49:00Z",
-  "access": {
-    "response_code": 404,
-    "method": "GET",
-    "url": "/foo/bar?q=hello"
-  }
-}
-```
-
-### Advice for particular frameworks or platforms
-
-#### Dropwizard
-
-The
-[dropwizard-logstash](https://github.com/alphagov/dropwizard-logstash)
-library will set things up for you using the standard names.
-
-#### Cloud Foundry
-
-The
-[GOV.UK PaaS Logging](https://docs.cloud.service.gov.uk/monitoring_apps.html#set-up-the-logit-io-log-management-service)
-documentation will help you configure logit and drain logs into it
-from your app.
+### Stuctured logging with Splunk
+
+[Splunk] automatically parses JSON log lines. Other formats may need [specific
+field extracts] configured in Splunk.
+
+For interoperability with pre-built apps and alerting, it is beneficial to align
+your logs to the [Splunk CIM (Common Information Model)].
+
+The [`Web` CIM] is most commonly used at GDS - it specifies particular field
+names for data, for example:
+
+  - `http_user_agent` for the client's user agent
+  - `src` for the source IP address
+  - `dest` for the origin server IP address
+
+## Access to Splunk
+
+Access control for GDS users is managed by the IT Service Desk, use the
+[helpdesk] to request access. If you're unsure what role you should be
+requesting, ask in the `#cyber-security-help` Slack channel.
+
+## Advice for particular frameworks or platforms
+
+### Dropwizard
+
+The [dropwizard-logstash] library will set things up for you using the
+standard names. Splunk will be able to ingest this format but the ingestion
+route may differ as Splunk Cloud does not support syslog inputs.
```

Comment:
I think this might be wrong as `dropwizard-logstash` is for formatting logs in a logstash format. Nothing to do with syslog. Which is a transport protocol not a log format.

Reply:
@gidsg - I've amended this paragraph and moved the syslog note elsewhere.